pudb/shell.py

```diff
@@ -96,12 +96,19 @@ def have_ipython():
     # https://github.com/ipython/ipython/issues/9435
 
     try:
+        # Assume don't have IronPython unless this variable is
+        # defined, see:
+        # http://stackoverflow.com/questions/5376837/run-from-ipython
+        # this skips the pain of demandimporters when IPython
+        # is available but not actually active:
+        # https://bz.mercurial-scm.org/show_bug.cgi?id=5346
+        __IPYTHON__
```

Comment:
My understanding is that this variable exists iff IPython has been imported. Wouldn't `'IPython' in sys.modules` be an equivalent check?